app/lib/common/widgets/member_list_entry.dart

```diff
@@ -239,13 +238,13 @@ class MemberListEntry extends ConsumerWidget {
                         style: Theme.of(context).textTheme.titleSmall,
                       ),
                       actions: <Widget>[
-                        DefaultButton(
+                        ElevatedButton(
                           onPressed: () {
                             // close both dialogs
                             context.pop();
                             context.pop();
                           },
-                          title: 'Okay',
+                          child: const Text('Okay'),
```

Comment:
Was thinking if there's a way to adapt font size based on devices, it's crucial based on user design perspective .i.e. people with disabilities would like to have shown larger font size than usual. Something like set up clamp on text scale factor would work well.